mtp_bank_admin/templates/base.html

```diff
@@ -2,40 +2,13 @@
 {% load i18n %}
 {% load mtp_common %}
 
+
+{% block page_title %}{% trans 'Bank admin' %}{% endblock %}
+
+
 {% block body_end %}
   <script src="{% url 'js-i18n' %}?{{ APP_GIT_COMMIT_SHORT }}"></script>
   {{ block.super }}
   {% sentry_js %}
   <!-- {{ request.resolver_match.url_name }} -->
 {% endblock %}
-
```

Comment:
Sorry if it's obvious, but I'm not quite sure what this `proposition_extra` was TBH. I'm guessing there was this tab before but we don't need anymore because we only have the main Dashboard page?

Oh, these were moved in the footer instead, isn't it? Must be...

Reply:
the `proposition_extra` was a block in the previous base template to be able to put something into "proposition" part of the header. that's what GDS called the place where the name of the service went.

we used it for the tabs you see at the bottom of the header in staff-only apps. there's an equivalent now here
https://github.com/ministryofjustice/money-to-prisoners-bank-admin/pull/302/files/f29c6799803bb4c89985879349d0d0c1c51172c3#diff-3fa72f0415f1c0a859fecedd2e28076ed590db4d934acc819aa51c53a1b6db0fR1